src/api/structures/member.ts

```diff
@@ -156,6 +156,8 @@ export interface Member {
   premiumType?: number;
   /** The guild related data mapped by guild id */
   guilds: Collection<string, GuildMember>;
+  /** whether the user has not yet passed the guild's Membership Screening requirements */
+  pending?: boolean;
```

Comment:
This should not be there since members are cached separately from guilds. Means if this is true the bot would think the member is pending on all guilds he is in. The pending parameter should be set in GuildMember (`member.guilds`)

Reply:
This should be moved to GuildMember